modules/rhel-compute-requirements.adoc

```diff
@@ -19,13 +19,13 @@ ifdef::openshift-origin[]
 ** Base OS: CentOS 7.4.
 endif::[]
 ifdef::openshift-enterprise,openshift-webscale[]
-** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/installation_guide/index[{op-system-base} 7.9] with "Minimal" installation option.
+** Base OS: link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/installation_guide/index[{op-system-base} 7.9] or link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[{op-system-base} 8.2 and later] with "Minimal" installation option.
 +
 [IMPORTANT]
 ====
 Adding {op-system-base} 7 compute machines to an {product-title} cluster is deprecated. Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
 
-In addition, you must not upgrade your compute machines to {op-system-base} 8 because support is not available in this release.
+In addition, you cannot upgrade your {op-system-base} 7 compute machines to {op-system-base} 8. You must deploy new {op-system-base} 8 hosts, and the old {op-system-base} 7 hosts should be removed.
```

Comment:
Could we also add a link to the steps for removing workers from OCP cluster, that would be helpful for users.

Reply:
Since we cannot provide cross-reference links in modules, I added a link to the [Deleting nodes](https://docs.openshift.com/container-platform/4.8/nodes/nodes/nodes-nodes-working.html#deleting-nodes) section just after this module.